pom.xml

```diff
@@ -82,11 +81,13 @@ under the License.
 		<module>flink-mesos</module>
 		<module>flink-metrics</module>
 		<module>flink-yarn</module>
+		<module>flink-shaded-yarn-tests</module>
 		<module>flink-yarn-tests</module>
 		<module>flink-fs-tests</module>
 		<module>flink-docs</module>
 		<module>flink-python</module>
 		<module>flink-ml-parent</module>
+		<module>flink-walkthroughs</module>
```

Comment:
I don't know why the pom diff is so crazy, I think it's because this is based on #8873 which does not have the most recent pom. I've verified that this line is the only difference between this branch and master.

Reply:
Rebased on fixed.